atlasdb-dbkvs-tests/src/test/java/com/palantir/nexus/db/pool/HikariCpConnectionManagerTest.java

```diff
@@ -67,35 +63,42 @@ public void testCanGetConnection() throws SQLException {
     @Test
     public void testCantGetConnectionIfClosed() throws SQLException {
         manager.close();
-        thrown.expect(SQLException.class);
-        thrown.expectMessage("Hikari connection pool already closed!");
-        try (Connection conn = manager.getConnection()) {
-            fail("fail");
-        }
+        assertThatThrownBy(() -> {
+                    try (Connection connection = manager.getConnection()) {
+                        assertThat(connection).isNotNull();
```

Comment:
This is a bit strange - are we asserting that the `manager.getConnection` method fails, but at the same time want to clean up the resources? Can we rewrite this some other way?